dataactbroker/handlers/jobHandler.py

```diff
@@ -32,9 +32,14 @@ def getSubmissionsByUserAgency(self,user,limit=5):
         """ Returns all submissions associated with the specified user's agency """
         return self.session.query(Submission).filter(Submission.cgac_code == user.cgac_code).order_by(Submission.updated_at.desc()).limit(limit).all()
 
-    def getSubmissionsByUserId(self,userId):
+    def getSubmissionsByUserId(self,userId, limit, offset, certified):
         """ Returns all submissions associated with the specified user ID """
-        return self.session.query(Submission).filter(Submission.user_id == userId).all()
+        if certified is None:
+            return self.session.query(Submission).filter(Submission.user_id == userId).order_by(
+                Submission.updated_at.desc()).limit(limit).offset(offset).all()
+        else:
+            return self.session.query(Submission).filter(and_(Submission.user_id == userId, Submission.publishable == certified)).order_by(
```

Comment:
I don't think it's casting. Looks to be working as intended (based on the results i'm getting back) and I believe we're using string values elsewhere also.